src/main.js

```diff
@@ -11,18 +11,22 @@ Vue.use(VueAxios, axios);
 axios.defaults.baseURL = "https://developers.zomato.com/api/v2.1";
 axios.defaults.headers.common["user-key"] = process.env.VUE_APP_API_KEY;
 
-//interceptors
-
 axios.interceptors.response.use(
   response => {
-    if (response.status === 403) {
-      console.log(response);
+    if (process.env.VUE_APP_API_KEY) {
+      console.log(response.status);
     }
     return response;
   },
   error => {
-    if (error.response.status === 403) {
-      alert("please use your API key to run the app");
+    if (!process.env.VUE_APP_API_KEY) {
+      alert(
+        "Please make a file named '.env.local'. Create a global variable VUE_APP_API_KEY and initialize it with your own key in this file. " +
+          error
+      );
+    }
+    if (process.env.VUE_APP_API_KEY && error.response.status === 403) {
```

Comment:
What about other error codes? Are there any other kinds of errors you think you could handle?